lib/mix/tasks/phx.gen.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Mix.Tasks.Phx.Gen do
+  use Mix.Task
+
+  @shortdoc "Explains in detail what every phx.gen task do"
```

Comment:
```suggestion
  @shortdoc "Lists all available Phoenix generators"
```

Reply:
I use hexdocs a lot and considering the mix tasks help lists all the commands alphabetically, I had to read every generator one by one to realize how many of them do the same as the previous one plus a little bit more (which is awesome)

so i though, i should improve the documentation (in fact, now i think i should use a markdown table showing the relationship between the CRUD related tasks) and when I went for it I realized I needed to modify Phoenix tasks to achieve it. and considering this is a cross-cutting bit of information, I ended up suggesting this tasks.

In that sense, your idea of automating the listing of tasks is great, I would only add a table describing the CRUD related tasks by order of complexity. will commit them in a minute.